spring-restdocs-core/src/main/java/org/springframework/restdocs/JUnitRestDocumentation.java

```diff
@@ -34,12 +34,10 @@
 
 	/**
 	 * Creates a new {@code JUnitRestDocumentation} instance that will generate snippets
-	 * to the given {@code outputDirectory}.
-	 *
-	 * @param outputDirectory the output directory
+	 * to &lt;gradle/maven build path&gt;/generated-snippet if no runtime property ({@code snippetOutputDirectory}) is set.
 	 */
-	public JUnitRestDocumentation(String outputDirectory) {
```

Comment:
This constructor needs to be kept, both to allow the output directory to be overridden and to maintain backwards compatibility.